packages/@aws-cdk/cli-lib/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@aws-cdk/cli-lib",
+  "description": "AWS CDK Programmatic CLI library",
+  "version": "0.0.0",
+  "private": true,
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "awslint": "cdk-awslint",
+    "build": "cdk-build",
+    "build+extract": "yarn build && yarn rosetta:extract",
+    "build+test": "yarn build && yarn test",
+    "build+test+extract": "yarn build+test && yarn rosetta:extract",
+    "build+test+package": "yarn build+test && yarn package",
+    "compat": "cdk-compat",
+    "lint": "cdk-lint",
+    "package": "cdk-package",
+    "pkglint": "pkglint -f",
+    "rosetta:extract": "yarn --silent jsii-rosetta extract",
+    "test": "cdk-test",
+    "watch": "cdk-watch"
+  },
+  "awslint": {
+    "exclude": []
+  },
+  "cdk-build": {
+    "env": {
+      "AWSLINT_BASE_CONSTRUCT": true
+    }
+  },
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com",
+    "organization": true
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@aws-cdk/node-bundle": "0.0.0",
+    "@aws-cdk/cdk-build-tools": "0.0.0",
+    "@aws-cdk/pkglint": "0.0.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^14.18.32",
+    "ts-node": "^10.2.1",
+    "jest": "^27.5.1"
+  },
+  "dependencies": {
+    "aws-cdk": "0.0.0",
```

Comment:
If you turn `aws-cdk` into a `devDependency` and bundle it using a build script, it should be jsii-able.